docs/content/tutorial/saving-your-data.mdx

```diff
@@ -167,12 +268,25 @@ Despite the changes in where the data is stored, the developer experience hasn
 
 ---
 
+## When should you use I/O managers?
+
+I/O managers are a powerful tool that can simplify your code and allow you to easily modify where your assets are stored. In this tutorial, using an I/O manager made sense because the contents of each asset are used in the downstream assets. For example, you needed to load the `topstory_ids` asset in order to compute the `topstories` asset. Using an I/O manager allowed up to remove the repetitive reading and writing code.
+
+However, some assets do not need an I/O manager. For example, you may write an asset that executes a Snowflake query that creates a new tables based on the data of another table. This asset would depend on the the existing table, but does not need to load that table in memory in order to execute the query:
```

Comment:
```suggestion
However, some assets do not need an I/O manager. For example, you may write an asset that executes a Snowflake query that creates a new table based on the data of another table. This asset would depend on the existing table, but does not need to load that table in memory in order to execute the query:
```

Fixes some typos.

Also, should we link out to the docs for `deps` here or nah?